schema.graphql

```diff
@@ -1,33 +1,83 @@
 type VotingEscrow @entity {
+  " VotingEscrow contract address "
   id: ID!
+
+  " Amount of B-80BAL-20WETH BPT locked "
   stakedSupply: BigDecimal!
+
+  " List of veBAL locks created "
   locks: [VotingEscrowLock!] @derivedFrom(field: "votingEscrowID")
 }
 
 type VotingEscrowLock @entity {
+  " Equal to: <userAdress>-<votingEscrow> "
   id: ID!
+
+  " Reference to User entity "
   user: User!
+
+  " Timestamp at which B-80BAL-20WETH BPT can be unlocked by user [seconds] "
   unlockTime: BigInt
+
+  " Amount of B-80BAL-20WETH BPT the user has locked "
   lockedBalance: BigDecimal!
+
+  " Reference to VotingEscrow entity "
   votingEscrowID: VotingEscrow!
 }
 
 type GaugeFactory @entity {
+  " Factory contract address "
   id: ID!
+
+  " Number of gauges created through the factory "
   numGauges: Int!
+
+  " List of gauges created through the factory "
   gauges: [LiquidityGauge!] @derivedFrom(field: "factory")
 }
 
 type LiquidityGauge @entity {
+  " LiquidityGauge contract address "
   id: ID!
+
+  " ERC20 token symbol "
   symbol: String!
+
+  " Reference to Pool entity "
+  pool: Pool!
+
+  " Address of the pool (lp_token of the gauge) "
   poolAddress: Bytes!
+
+  " Pool ID if lp_token is a Balancer pool; null otherwise "
   poolId: Bytes
+
+  " Whether Balancer DAO killed the gauge "
   isKilled: Boolean!
-  streamer: Bytes # ChildChainLiquidityGauge Only
+
+  " Whether Balancer DAO added the gauge to GaugeController "
+  isAdded: Boolean!
+
+  " Timestamp at which Balancer DAO added the gauge to GaugeController [seconds] "
```

Comment:
" Timestamp at which Balancer DAO added the gauge to GaugeController [seconds] - only applies to mainnet gauges "